tests/test_caplugin.py

```diff
@@ -159,6 +159,46 @@ async def test_get_enum_pv(engine: Engine, ioc: Popen):
     )
 
 
+@pytest.mark.asyncio
+async def test_subscribe_disconnect(engine: Engine, ioc: Popen):
+    query = (
+        """
+subscription {
+    subscribeChannel(id: "ca://%slongout") {
+        value {
+            float
+        }
+        status {
+            quality
+        }
+    }
+}
+"""
+        % PV_PREFIX
+    )
+    results = []
```

Comment:
Why does Mypy require a type hint here? Why is it even checking this function? It seems entirely equivalent to `test_subscribe_ticking()`.